core/state_update_test.go

```diff
@@ -12,33 +13,37 @@ import (
 )
 
 func TestStateDiffCommitment(t *testing.T) {
-	client := feeder.NewTestClient(t, &utils.Integration)
+	client := feeder.NewTestClient(t, &utils.Mainnet)
```

Comment:
Why do we replace some Goerli/Integration tests with Mainnet tests, I would have expected them to use Sepolia data

Reply:
Otherwise it looks good

Reply:
My reasoning for such data based tests was to draw more on the mainnet data which likely not get off soon (so there will be no need to change the test then).